code-bar-job-board-app/src/components/SubmitJob/index.js

```diff
@@ -23,9 +24,17 @@ const SumbitJobPage = ({createJobPost, currentUser}) => {
 
 
     const handleJobPostSubmitForm = (evt) => {
+        const form = evt.currentTarget;
+        if (form.checkValidity() === false) {
+            evt.preventDefault();
+            evt.stopPropagation();
```

Comment:
Tested this and it shows validation fine, but still sends to the DB.  I think a return statement here would stop that.